eos_data_distribution/avahi_monitor.py

```diff
@@ -31,11 +32,19 @@
 from eos_data_distribution.names import SUBSCRIPTIONS_SOMA
 from eos_data_distribution.MDNS import ServiceDiscovery
 
+
 SERVICES = [
     # Disable TCP, we really only want UDP or ethernet
     # "_nfd._tcp",
     "_nfd._udp"]
 
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def flatten(l):
+    return [i for s in l for i in s]
```

Comment:
nit: I think `list(itertools.chain.from_iterable(l))` is cleaner, but no big deal